etl/steps/data/garden/countries/2023-09-25/gleditsch.meta.yml

```diff
@@ -24,31 +38,9 @@ tables:
         description_short: |-
           Number of states (and microstates) in each region.
```

Comment:
again maybe something more informative for the description_short here? `Number of states and microstates in each region reflects their status as sovereign states, defined by their relatively autonomous administration of territory and recognition of independence by local actors and other states.`

Reply:
this is already captured in `definitions.common.description_key`